progression/src/main/java/me/mykindos/betterpvp/progression/profession/woodcutting/WoodcuttingLoadConfig.kt

```diff
@@ -0,0 +1,72 @@
+package me.mykindos.betterpvp.progression.profession.woodcutting
+
+import me.mykindos.betterpvp.core.config.ExtendedYamlConfiguration;
+import me.mykindos.betterpvp.core.logging.CustomLogger
+import me.mykindos.betterpvp.core.utilities.model.WeighedList
+import org.bukkit.Material
+import org.bukkit.configuration.ConfigurationSection
+import java.util.*
+
+
+/**
+ * - Loads the YAML configuration for the Woodcutting Profession & returns the loaded data
+ * - This function will cause side effects by logging messages to the console
+ */
+fun loadConfigForWoodcutting(config: ExtendedYamlConfiguration,
```

Comment:
Any particular reason you did this in kotlin? we have plenty of examples in java of loading config this way

Reply:
I thought the code would be more concise if put in Kotlin but I can change it to Java if want me to

Reply:
To keep it consistent with the other 99.9% of the project, its probably best to keep it in java